src/runtime/alignment_128.cpp

```diff
@@ -1,13 +1,5 @@
 #include "runtime_internal.h"
 
-namespace Halide {
-namespace Runtime {
-namespace Internal {
-
-WEAK_INLINE int halide_malloc_alignment() {
+extern "C" WEAK_INLINE int halide_malloc_alignment() {
```

Comment:
suggestion: rename these calls `halide_internal_foo()` to preserve the 'internalness' of them in the name? (no one externally should be overriding them anyway so theoretically this shouldn't break anything.)

Reply:
+1.  I think it'll be better in the long run to easily be able to see at a glance it's internal.

Reply:
There's another convention used elsewhere to prefix with a leading underscore, is that for signaling internal?

Reply:
Yes, we have three active conventions for internal stuff:

1) Halide::Runtime::Internal
2) _halide
3) Just call it halide_ (most of the existing stuff in runtime_internal.h was in this category)

In this PR I deduped 1 into 3. I could further dedup 3 into 2, but that would be a much larger change and I would prefer to do it in a non-bugfix PR.

Reply:
SGTM